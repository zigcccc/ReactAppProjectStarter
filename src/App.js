import React from 'react';

/**
 * Root App Component
 * @returns {StatelessComponent}
 */
const App = () => <div className="app">New React App</div>;

export default App;
