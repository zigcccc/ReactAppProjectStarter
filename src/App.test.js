import { shallow } from 'enzyme';
import React from 'react';

import App from './App';

test('renders root App component without crashing', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
