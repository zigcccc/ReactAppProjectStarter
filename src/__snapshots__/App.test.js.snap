// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders root App component without crashing 1`] = `
<div
  className="app"
>
  New React App
</div>
`;
